odl/space/cu_ntuples.py

```diff
@@ -668,6 +676,39 @@ def exponent(self):
         """Exponent of the norm and distance."""
         return self._space_funcs.exponent
 
+    @property
+    def weighting(self):
```

Comment:
Done that. One slight remark about this is that `FnBase` does not guarantee existence of a `weighting` attribute, even if both its implementations do. I wrote the properties with `gettattr` and default `None` and `False`, resp.